<?xml version="1.0"?>
<launch>
    <param name="/octomap_topic_name" value="/global_planner_node/octomap_full"/>
<!--    <param name="/octomap_topic_name" value="/mbplanner_node/octomap_full"/>-->
    <param name="/octomap_resolution" value="0.3"/>
    <node name="grapher" pkg="octomap_grapher" type="octomap_grapher_node" output="screen"/>
</launch>
